### PR TITLE
New package: Posit.rig version 0.5.3

### PR DESCRIPTION
**Pull request by eitsupi**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108337)

### CODE REVIEW

File: manifests/p/Posit/rig/0.5.3/Posit.rig.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-04-20
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/p/Posit/rig/0.5.3/Posit.rig.installer.yaml:16]:
```suggestion
- Architecture: x64
```
IMG_0306

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
RIG -- The R Installation Manager 0.5.3
NAME
    rig - manage R installations

DESCRIPTION
    rig manages your R installations, on macOS, Windows, and Linux. It can
    install and set up multiple versions of R, and make sure that they work
    together.

    rig is currently experimental and is a work in progress. Feedback is much
    appreciated. See https://github.com/r-lib/rig for bug reports.
...
```

Also, the Start menu shortcut appears to do nothing - a window pops up and immediately closes. Is that expected?

**Comment by eitsupi** [comment]
Hi @stephengillie, thanks for your review!

> This package is expecting arguments:

It is not clear to me what you mean by this point of yours.
Do I need to update the manifest to state something?

> Also, the Start menu shortcut appears to do nothing - a window pops up and immediately closes. Is that expected?

This is unrelated behavior to winget, but as you point out, I don't think it's ideal.
I created an issue r-lib/rig#160.
In other words, this is the expected behavior for this version.

**Comment by stephengillie** [comment]
> Hi @stephengillie, thanks for your review!
>
> > This package is expecting arguments:
> 
> It is not clear to me what you mean by this point of yours. Do I need to update the manifest to state something?

The package might need a `Command:` or similar field added to the manifest in the future. I haven't had an opportunity to learn the exact fix for this situation, but felt that notes like these might help to explain why the error label was removed or waivered. I didn't mean to disturb or insult, and can avoid posting these in your PRs if you would like. 

> > Also, the Start menu shortcut appears to do nothing - a window pops up and immediately closes. Is that expected?
> 
> This is unrelated behavior to winget, but as you point out, I don't think it's ideal. I created an issue [r-lib/rig#160](https://github.com/r-lib/rig/issues/160). In other words, this is the expected behavior for this version.

Thank you for confirming the expected behavior.

**Review by stephengillie — APPROVED** [review]

**Comment by eitsupi** [comment]
> The package might need a `Command:` or similar field added to the manifest in the future. I haven't had an opportunity to learn the exact fix for this situation, but felt that notes like these might help to explain why the error label was removed or waivered.

That makes sense.
I just wanted to know what needed to be changed and I don't care. Thanks for taking a look at this!

**Pull request merged**